src/components/profile-tab.yml

```diff
@@ -1,6 +1,6 @@
 info:
-  en: Here you can provide additional information about yourself. This information is currently not publicly accessible.
-  de: Hier können Sie zusätzliche Informationen zu Ihrer Person angeben. Die Daten sind zur Zeit nicht öffentlich sichtbar.
+  en: Here you can provide additional information about yourself. This information is publicly accessible through your <0>profile page</0>.
+  de: Hier können Sie zusätzliche Informationen zu Ihrer Person angeben. Die Daten sind auf Ihren <0>Profilseite</0> öffentlich sichtbar.
```

Comment:
Ihre**r**

Reply:
**Die** Seite, of course 🤦‍♀️ Such basic mistakes.. Corrected, thank you.